cypress/integration/ui/view_dashboard.feature

```diff
@@ -54,10 +54,17 @@ Feature: Viewing dashboards
         When I toggle the control bar height
         Then the control bar should be expanded to full height
 
+# TODO: flaky test
```

Comment:
Is this something to fix?

Reply:
yes, I need to take a look at all the flaky tests and figure out how to make them reliable.